docs/about/syntaxes.md

```diff
@@ -0,0 +1,19 @@
+# Syntaxes
+
+There are many styling languages, ranging from CSS language extension like SCSS to entirely different notations like CSS-in-JS objects. These styling languages can be then be embedded within other languages, including HTML `<style>` tags, markdown fences and JavaScript variables.
+
+We aim to support all these use cases, but it's a complicated task.
+
+We use [PostCSS syntaxes](https://github.com/postcss/postcss#syntaxes) to transform these languages into something that resembles CSS, which is the language that:
+
+-   underpins all the other styling languages
+-   is best understood by stylelint
+
+We need your help to [support and improve](https://github.com/postcss/postcss/blob/master/docs/syntax.md) the following PostCSS syntaxes:
+
+-   postcss-html
+-   postcss-jsx
+-   postcss-less
+-   postcss-markdown
+-   postcss-sass
+-   postcss-scss
```

Comment:
[suggestion] What about adding links to each repository like <https://github.com/postcss/postcss#syntaxes>?

<img width="392" alt="image" src="https://user-images.githubusercontent.com/473530/73473631-6e3d6f80-43d0-11ea-93a2-2c8dd75705ad.png">

Reply:
Yes, this is a good idea now that we're starting to fork the syntaxes we use into the stylelint org.